lib/node_modules/@stdlib/blas/ext/base/sdssum/manifest.json

```diff
@@ -24,19 +28,153 @@
 	],
 	"confs": [
 		{
+			"os": "linux",
+			"blas": "",
+			"wasm": false,
+			"src": [
+				"./src/sdssum.f",
+				"./src/sdssumsub.f",
+				"./src/sdssum_f.c"
+			],
+			"include": [
+				"./include"
+			],
+			"libraries": [],
+			"libpath": [],
+			"dependencies": [
+				"@stdlib/napi/export",
+				"@stdlib/napi/argv",
+				"@stdlib/napi/argv-int64",
+				"@stdlib/napi/argv-strided-float64array"
+			]
+		},
+		{
+			"os": "linux",
+			"blas": "openblas",
+			"wasm": false,
+			"src": [
+				"./src/sdssum_cblas.c"
+			],
+			"include": [
+				"./include"
+			],
+			"libraries": [
+				"-lopenblas",
+				"-lpthread"
+			],
+			"libpath": [],
+			"dependencies": [
+				"@stdlib/napi/export",
+				"@stdlib/napi/argv",
+				"@stdlib/napi/argv-int64",
+				"@stdlib/napi/argv-strided-float64array"
+			]
+		},
+		{
+			"os": "mac",
+			"blas": "",
+			"wasm": false,
+			"src": [
+				"./src/sdssum.f",
+				"./src/sdssumsub.f",
+				"./src/sdssum_f.c"
+			],
+			"include": [
+				"./include"
+			],
+			"libraries": [],
+			"libpath": [],
+			"dependencies": [
+				"@stdlib/napi/export",
+				"@stdlib/napi/argv",
+				"@stdlib/napi/argv-int64",
+				"@stdlib/napi/argv-strided-float64array"
+			]
+		},
+		{
+			"os": "mac",
+			"blas": "apple_accelerate_framework",
+			"wasm": false,
+			"src": [
+				"./src/sdssum_cblas.c"
+			],
+			"include": [
+				"./include"
+			],
+			"libraries": [
+				"-lblas"
+			],
+			"libpath": [],
+			"dependencies": [
+				"@stdlib/napi/export",
+				"@stdlib/napi/argv",
+				"@stdlib/napi/argv-int64",
+				"@stdlib/napi/argv-strided-float64array"
+			]
+		},
+		{
+			"os": "mac",
+			"blas": "openblas",
+			"wasm": false,
+			"src": [
+				"./src/sdssum_cblas.c"
+			],
+			"include": [
+				"./include"
+			],
+			"libraries": [
+				"-lopenblas",
+				"-lpthread"
+			],
+			"libpath": [],
+			"dependencies": [
+				"@stdlib/napi/export",
+				"@stdlib/napi/argv",
+				"@stdlib/napi/argv-int64",
+				"@stdlib/napi/argv-strided-float64array"
+			]
+		},
+		{
+			"os": "win",
+			"blas": "",
+			"wasm": false,
+			"src": [
+				"./src/sdssum.c"
+			],
+			"include": [
+				"./include"
+			],
+			"libraries": [
+				"-m"
+			],
+			"libpath": [],
+			"dependencies": [
+				"@stdlib/napi/export",
+				"@stdlib/napi/argv",
+				"@stdlib/napi/argv-int64",
+				"@stdlib/napi/argv-strided-float64array"
+			]
+		},
+		{
+			"os": "",
+			"blas": "",
+			"wasm": true,
 			"src": [
 				"./src/sdssum.c"
 			],
 			"include": [
 				"./include"
 			],
 			"libraries": [
-				"-lm"
+				"-m"
 			],
 			"libpath": [],
 			"dependencies": [
-				"@stdlib/blas/ext/base/sdssumpw"
+				"@stdlib/napi/export",
+				"@stdlib/napi/argv",
+				"@stdlib/napi/argv-int64",
+				"@stdlib/napi/argv-strided-float64array"
 			]
 		}
 	]
-}
+}
```

Comment:
Trailing newline. Please ensure you have installed EditorConfig, as mentioned in the development guide.